lib/src/Base/Stat/csv_lexer.ll

```diff
@@ -43,7 +43,7 @@ integer   {digit}+
 exponent  [eE]{sign}?{integer}
 real      {sign}?({integer}("."({integer})?)?|"."{integer}){exponent}?
 
-alpha     [a-zA-Z_]
+alpha     [a-zA-Z_()]
 string    {alpha}({alpha}|{integer})*
```

Comment:
This seems restrictive still a bit restrictive compared to the autorized chars in the norm:
https://tools.ietf.org/html/rfc4180#page-2

it mentions TEXTDATA =  %x20-21 / %x23-2B / %x2D-7E
that is [space]! / #$%&'()*+ / -./[0-9]:;<=>?@[A-Z][\]^_`[a-z]{|}
according to https://fr.wikipedia.org/wiki/American_Standard_Code_for_Information_Interchange
so that would mean these characters should also be allowed.